sparkle/src/components/SearchInput.tsx

```diff
@@ -118,6 +119,7 @@ export const SearchInputWithPopover = forwardRef<
       value,
       onChange,
       mountPortal = false,
```

Comment:
Let's not set any default value for `mountPortal`?

Reply:
👍 
it will use default from popover then (true)